### PR TITLE
issue 348 @testable import

### PR DESCRIPTION
**Pull request by Kharchevskyi**
This PR adds FlowCryptAppTest to use `@testable import FlowCrypt`

Part of #348

Pros: 
* Will compile code only once. Now files are compiling twice (Once for FlowCrypt and one for FlowCryptTests)
* Will run tests faster
* No need to add files to test targets.
Screenshot 2021-06-18 at 12 27 45

Before
Screenshot 2021-06-18 at 12 17 21

* Tests will run faster on CI.
* Easier to test. For instance if your need to test some `.swift` file all recursively dependent class/structs should be also added to test target.
If this is ok - will move test one by one to a new target, then old target will be removed.

**Tests**
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: FlowCryptAppTests/GmailSearchExpressionGeneratorTests.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import XCTest
+@testable import FlowCrypt
```
Comment (Kharchevskyi) [inline FlowCryptAppTests/GmailSearchExpressionGeneratorTests.swift:10]: `GmailSearchExpressionGenerator` is only in FlowCrypt target

**Comment by ekievsky** [comment]
It would be great to merge this PR by Monday so I could include this feature into my PR and create the tests.

**Comment by Kharchevskyi** [comment]
@tomholub if you ok with this solution I will proceed with this and finish probably today or tomorrow

**Review by tomholub — DISMISSED** [review]
Sure, this looks good.

**Review by tomholub — COMMENTED** [review]
So far no issues

**Comment by Kharchevskyi** [comment]
- [x] Configured schemes to run tests on CI
- [x] Removed old test target
- [x] Moved all existed tests to a new Target

**Review by tomholub — APPROVED** [review]

**Pull request merged**